package/MDAnalysis/analysis/rms.py

```diff
@@ -380,7 +380,10 @@ def __init__(self, atomgroup, reference=None, select='all',
              corresponding atom in `select`, and assumes ``None`` for `groupselections`.
 
         weights_groupselections : False or list of {"mass", ``None`` or array_like} (optional)
-             1. ``False`` will apply imposed weights to `groupselections` from ``weights`` option.
+             1. ``False`` will apply imposed weights to `groupselections` from
+             ``weights`` option if `weights` is not iterable. Otherwise will assume
```

Comment:
Thanks for making that change, @z3y50n. Could you actually make this a little more detailed, though -- it's not intended to be just any iterable, but values of None or "mass". The current message could imply that users can pass in any old value.

Reply:
Also, I think this is your first pull request with us -- please add yourself to the AUTHORS and CHANGELOG :-)

Reply:
So from what I understand it should say something like `if weights is not of the appropriate value`? Because in the [code](https://github.com/z3y50n/mdanalysis/blob/4ea0b6f244a7ac572d9fbf1e208ffbf27734a1da/package/MDAnalysis/analysis/rms.py#L577) it just checks if `weights` is `iterable`

Reply:
> Because in the code it just checks if weights is iterable

@z3y50n that's true, but consider what kind of non-iterable weights are valid for the code that [executes immediately after](https://github.com/z3y50n/mdanalysis/blob/4ea0b6f244a7ac572d9fbf1e208ffbf27734a1da/package/MDAnalysis/analysis/rms.py#L582-L589). Does it make sense to assign an array of weights that are all the same float?